pkg/cmd/space/delete/delete.go

```diff
@@ -32,26 +33,28 @@ func NewCmdDelete(f factory.Factory) *cobra.Command {
 
 			itemIDOrName := args[0]
 
-			client, err := f.GetSystemClient(apiclient.NewRequester(cmd))
+			octopus, err := f.GetSystemClient(apiclient.NewRequester(cmd))
```

Comment:
rename to avoid name clash with 'client' package. Also we tend to call this 'octopus' everywhere else too